pkg/release/release.go

```diff
@@ -187,6 +188,8 @@ func (r *Release) Install(ctx context.Context, name string, version Version, val
 		}
 	}
 
+	r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("ensured creation of release %#q", name))
+
```

Comment:
This code isn't idempotent so `ensure` doesn't fit there. It should be regular `creating`. I in general think we should avoid logging inside libraries, but let's see.

Reply:
Why isn't it idempotent though? I can fix the logs meanwhile but the method should maybe be improved in the future, shouldn't it?

Reply:
I wouldn't say so. As we agreed in `fmt` recently idempotent version should be named `EnsureInstalled` so we know what to expect. I think it should provide better error handling rather than idempotency assuming we retain this name.

Reply:
But we just want to install something and make sure what we want to be installed is installed so having an actual idempotent version to just ensure the e2e setup is correct would make sense.

Reply:
Not sure. We expect releases to not be installed in the control plane when we call those functions. I.e. we want a "fresh installation". Also those functions are called only once per release per test, so idempotency wouldn't bring much value there.

Reply:
But still the function name doesn't reflect idempotency. The code isn't idempotent. When we make it idempotent changing two log messages is rather easy.

Reply:
The log message is changed.